AvatarChatbot/tests/test_compose_on_gaudi.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+echo "IMAGE_REPO=${IMAGE_REPO}"
```

Comment:
https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/tests/test_compose_on_gaudi.sh#L6-L11
Add this part make sure the `IMAGE_REPO` and `IMAGE_TAG` is controllable.